doc/locale/ja/LC_MESSAGES/reference/commands/select.po

```diff
@@ -3185,6 +3187,46 @@ msgstr ""
 "ません。この場合は ``_key`` を使ってください。これは、 :ref:`select-"
 "drilldown-output-columns` と同じルールです。"
 
+msgid ""
+"Specifies ``${LABLE}`` of other ``drilldown``, ``drilldowns`` or ``slices``."
+msgstr ""
+"他の ``drilldown``、 ``drilldowns`` または ``slices`` の ``${LABLE}`` を指定"
+"します。"
+
+msgid ""
+"The result of the specified ``${LABLE}`` is drilldowned, which means that "
+"this paramter enables nested aggregate calculations and groups in drilldown."
```

Comment:
If 多段ドリルダウン can be called simply "nested drilldown" as mentioned in line 3204.
It will better to keep that words together.

個々の日本語「つまり、このパラメータは多段ドリルダウンを可能にします。」ですが、英語から理解すると
→つまりこのパラメータは、aggregate calculations and groupsによって多段ドリルダウンを可能にします。
ということかと思ったので下記のようなほうがシンプルだと思います。

This parameter enables nested drilldown with aggregate calculations and groups.

Reply:
「このパラメーターを使えば多段ドリルダウンできます」とかもっとシンプルな言い回しにしませんか？